Dockerfile

```diff
@@ -3,26 +3,21 @@ MAINTAINER Tim Perry <pimterry@gmail.com>
 
 USER root
 
-RUN apt-get update && \
-    export DEBIAN_FRONTEND=noninteractive && \
-    # Set password to temp-password - reset to random password on startup
-    echo mysql-server mysql-server/root_password password temp-password | debconf-set-selections && \
-    echo mysql-server mysql-server/root_password_again password temp-password | debconf-set-selections && \   
-    # Instal MySQL for data, node as the JS engine for uglifier
-    apt-get install -y mysql-server nodejs
-    
-COPY . /opt/staytus
+ENV DEBIAN_FRONTEND="noninteractive"
 
-RUN cd /opt/staytus && \
-    bundle install --deployment --without development:test
-
-ENTRYPOINT /opt/staytus/docker-start.sh
+ADD docker-start.sh /docker-start.sh
 
-# Persists all DB state
-VOLUME /var/lib/mysql
+RUN chmod 755 docker-start.sh && \
+    apt-get update && \
+    # Instal node as the JS engine for uglifier
+    apt-get install -y nodejs && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
+    mkdir -p /opt/staytus && \
+    git clone https://github.com/adamcooke/staytus.git /opt/staytus && \
```

Comment:
It shouldn't be necessary to clone the repo here. Just `ADD . /opt/staytus`. (Cloning the repo like this will also make it impossible to build a docker image from any branch other than master.)